Tests/Data/ThermoHydroMechanics/Linear/Point_injection/square_1e2.prj

```diff
@@ -0,0 +1,288 @@
+<?xml version="1.0" encoding="ISO-8859-1"?>
+<OpenGeoSysProject>
+    <mesh axially_symmetric="true">quarter_002_2nd.vtu</mesh>
+    <geometry>square_1x1.gml</geometry>
+    <processes>
+        <process>
+            <name>THERMO_HYDRO_MECHANICS</name>
+            <type>THERMO_HYDRO_MECHANICS</type>
+            <integration_order>4</integration_order>
+            <dimension>2</dimension>
+            <constitutive_relation>
+                <type>LinearElasticIsotropic</type>
+                <youngs_modulus>E</youngs_modulus>
+                <poissons_ratio>nu</poissons_ratio>
+            </constitutive_relation>
+            <intrinsic_permeability>k</intrinsic_permeability>
+            <specific_storage>S</specific_storage>
+            <biot_coefficient>alpha</biot_coefficient>
+            <porosity>phi</porosity>
+            <solid_density>rho_sr</solid_density>
+			<fluid_density>rho_fr</fluid_density>
+			<fluid_viscosity>mu</fluid_viscosity>
+            <fluid_volumetric_thermal_expansion_coefficient>beta_f</fluid_volumetric_thermal_expansion_coefficient>
+            <solid_linear_thermal_expansion_coefficient>alpha_s</solid_linear_thermal_expansion_coefficient>
+            <solid_specific_heat_capacity>C_s</solid_specific_heat_capacity>
+            <solid_thermal_conductivity>lambda_s</solid_thermal_conductivity>
+            <fluid_specific_heat_capacity>C_f</fluid_specific_heat_capacity>
+            <fluid_thermal_conductivity>lambda_f</fluid_thermal_conductivity>
+            <reference_temperature>T0</reference_temperature>
+            <process_variables>
+                <displacement>displacement</displacement>
+                <pressure>pressure</pressure>
+                <temperature>temperature</temperature>
+            </process_variables>
+            <secondary_variables>
+                <secondary_variable type="static" internal_name="sigma" output_name="sigma"/>
+                <secondary_variable type="static" internal_name="epsilon" output_name="epsilon"/>
+            </secondary_variables>
+            <specific_body_force>0 0</specific_body_force>
+        </process>
+    </processes>
+    <time_loop>
+        <processes>
+            <process ref="THERMO_HYDRO_MECHANICS">
+                <nonlinear_solver>basic_newton</nonlinear_solver>
+                <convergence_criterion>
+                    <type>PerComponentDeltaX</type>
+                    <norm_type>NORM2</norm_type>
+                    <abstols>1e-5 1e-5 1e-5 1e-5</abstols>
+                </convergence_criterion>
+                <time_discretization><type>BackwardEuler</type></time_discretization>
+                <time_stepping>
+                    <type>FixedTimeStepping</type>
+                    <t_initial>0</t_initial>
+                    <t_end>1000</t_end>
+                    <timesteps>
+                        <pair><repeat>10</repeat>
+                        <delta_t>100</delta_t></pair>
```

Comment:
Documentation mentions time step size to be 10000s, (and no end time).

Reply:
The whole test in documentation takes too much time and the results are already clearly shown in the pdf.   Here is only part of the simulation compared with expected value like the test in thermomechanics and so on.

Reply:
@grubbymoon Maybe, you can say in the documentation, that the benchmark only simulates the first part (mention the time) of the whole setup?
Also, @endJunction's comment referred to time step size, not simulation time.

Reply:
In the documentation, the total simulation time is too long so I make the time step larger. While in the test, I only simulate the first part of the whole setup, so the time step is set to be smaller. @waltherm No problem, I will add some descriptions in the documentation.

Reply:
The point is that there is discrepancy between the pdf and the test.

Just being curious I run the simulation with dt=1e4 and end time 1e7 and it took just under 1000s wall clock time, so 20 minutes --- that's not much indeed.

Last iteration looks like this:
```
info: *** Eigen solver computation
info: -> scale
info: -> solve with SparseLU
info: ------------------------------------------------------------------
info: [time] Linear solver took 0.10459 s.
info: Convergence criterion, component 0: |dx|=1.8622e-11, |x|=4.0371e+04, |dx|/|x|=4.6127e-16
info: Convergence criterion, component 1: |dx|=6.8909e-05, |x|=5.8987e+07, |dx|/|x|=1.1682e-12
info: Convergence criterion, component 2: |dx|=8.8176e-17, |x|=9.6344e-03, |dx|/|x|=9.1522e-15
info: Convergence criterion, component 3: |dx|=9.7433e-17, |x|=9.5955e-03, |dx|/|x|=1.0154e-14
info: [time] Iteration #5 took 0.151682 s.
info: [time] Solving process #0 took 0.763729 s in time step #1000 
info: [time] Time step #1000 took 0.763746 s.
info: [time] Output of timestep 1000 took 0.0173891 s.
info: The whole computation of the time stepping took 1000 steps, in which
         the accepted steps are 1000, and the rejected steps are 0.

info: [time] Execution took 956.165 s
```

Reply:
The documentation is updated with the description of the test and the time step size are set to be the same.